scripts/docker-build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+GCLOUD_FILE="$1"
+GIT_COMMIT="$2"
+BRANCH_NAME="$3"
+
+BRANCH_TAG="gcr.io/netlify-services/github-netlify-build-image-plugins:${BRANCH_NAME//\//-}"
+COMMIT_TAG="gcr.io/netlify-services/github-netlify-build-image-plugins:$GIT_COMMIT"
+
+docker build -t "${BRANCH_TAG}" .
+docker tag ${BRANCH_TAG} ${COMMIT_TAG}
+
+if [ ! -d ${HOME}/google-cloud-sdk ]; then
+  curl https://sdk.cloud.google.com | bash;
+fi
+source ${HOME}/google-cloud-sdk/path.bash.inc
+
+gcloud auth activate-service-account --key-file "${GCLOUD_FILE}"
+
+gcloud docker -- push "${BRANCH_TAG}"
```

Comment:
This probably should be a separate PR, but I am thinking more and more that Build plugins should just get stored somewhere else than a Docker image, if this is feasible. This would solve the following issues:
  - Docker image becoming too big: https://github.com/netlify/ansible/issues/4062#issuecomment-645227796 ([Slack thread](https://app.slack.com/client/T02UKDKNA/CDV7MQR3Q/thread/CDV7MQR3Q-1592396754.254400))
  - Plugins location issues: https://github.com/netlify/buildbot/issues/821

Reply:
Yeah, that's a fair point. I agree we can look into it as a follow-up.

I think this will still be a step in the right direction, because we're going to copy these plugins into the final image rather than installing them there directly - so we'll need to think a bit about everything that's being installed and make sure it's all getting captured.